docs/upgrade/upgrade-security.asciidoc

```diff
@@ -1,44 +1,77 @@
 
 NOTE: You do not need to shut down your {agent}s or endpoints to upgrade the {stack}.
 
-*Disable all rules*
+*Requirements to display Data views in the {es-sec-app}*
 
-Manually disable your rules before upgrading, then reactivate them after the process completes. If this is not done, rules are automatically disabled during the upgrade process, which can cause a gap in rule coverage.
+To make the *Data view* option appear in an environment with legacy alerts, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), then refresh the page. The user's role privileges must allow them to enable the detections feature in a Kibana space. Refer to <<enable-detections-ui, Enable and access detections>> for more information.
 
-To disable all rules:
+NOTE: If new alerts are generated in an upgraded environment without legacy alerts, refreshing any page with alert data in {elastic-sec} will make the *Data view* option appear in the {es-sec-ui}.
 
-. Go to *Detect* -> *Rules*.
-. Click the *Select All Rules* button above the All rules table.
-. Click *Bulk actions* -> *Deactivate selected*.
+*New alert schema*
 
-To reactivate all detection rules:
+The system index for detection alerts has been renamed from `.siem-signals-<space-id>` to `.alerts-security.alerts-<space-id>` and is now a hidden index. Therefore, the schema used for alert documents in {elastic-sec} has changed. Users that access documents in the `.siem-signals` indices using the {elastic-sec} API must modify their API queries and scripts to operate properly on the new 8.0.0 alert documents. Refer to <<query-alert-indices, how to query alert indices>> and review the new <<alert-schema, Alert schema>>.
 
-. Go to *Detect* -> *Rules*.
-. Click the *Select All Rules* button above the All rules table.
-. Click *Bulk actions* -> *Activate selected*.
+*New privileges required to view alerts and preview rules*
 
-NOTE: To use the {elastic-sec} API to re-enable rules, refer to <<bulk-actions-rules-api>>.
+* To view alerts, users need `manage`, `write`, `read`, and `view_index_metadata` privileges to two new indices, `.alerts-security.alerts` and `.internal.alerts-security.alerts`. Existing users who are upgrading to 8.0.0 can retain their privileges to the `.siem-signals` index.
 
-*New Full Disk Access approval needed for {elastic-endpoint}*
+* To <<preview-rules, preview rules>>, users need `read` access to the new `.preview.alerts-security.alerts` index. Refer to <<detections-permissions-section>> for more information.
+
+*Full Disk Access (FDA) approval for {elastic-endpoint}*
+
+New FDA approval is needed when deploying {elastic-endpoint} on macOS. Refer to <<deploy-elastic-endpoint, Install {elastic-endpoint} manually>> for more information.
```

Comment:
This section covers the same information in the previous section "New Full Disk Access approval needed for {elastic-endpoint}" (starting on line 40 above). Remove one or the other? This one seems to have the least info, so maybe remove it and keep the one above.